snippets/facets.liquid

```diff
@@ -71,13 +71,13 @@
                               {% if value.count == 0 and value.active == false %}disabled{% endif %}
                             >
 
-                            <svg width="16" height="16" viewBox="0 0 16 16" aria-hidden="true" focusable="false">
+                            <svg width="1.6rem" height="1.6rem" viewBox="0 0 16 16" aria-hidden="true" focusable="false">
```

Comment:
Wondering if we'll need to do this to all of the other `svgs`?

Reply:
While I'm not sure if it should apply to all `svg`s, it should probably apply consistently to icons that are accompanied by text.

For instance, on the product page, the pickup availability one follows the text size, but the collapsible tabs and share button does not.

![image](https://user-images.githubusercontent.com/37168033/135142065-f56414ad-af1c-4c31-bfb9-7846a75aea33.png)

Reply:
Updated https://github.com/Shopify/dawn/pull/688/commits/2ff7cac0ff3b224b861db3ade8f517e66f94bb50:

- Share icon now scales with `body_scale` setting
- Collapsible tab heading icons now scale with `heading_scale` setting